examples/amass_human_poses/README.md

```diff
@@ -0,0 +1,116 @@
+
+However, we will give you a step by step explanation on how to get access.
+
+1. Visit the following websites, sign up as user after you agree to the terms of use.
+	1. `https://amass.is.tue.mpg.de/`
+	2. `https://mano.is.tue.mpg.de/`
+	3. `https://smpl.is.tue.mpg.de/`
+2. wait till receive the confirmation vie e-mail, once you confirm your account you can access download sections.
+3. download the following files which represent the body models and extract them inside folder `resources/AMASS/body_models`
+	1. from `https://smpl.is.tue.mpg.de/downloads` download the following file
+
+	<img src="instructions_screenshots/smpl_model.png" width=350> 
+	
+	2. from `https://mano.is.tue.mpg.de/downloads`download the following file 
+
+	<img src="instructions_screenshots/dmpl_model.png" width=350> 
+	
+5. explore the collection of mation capture dataset that are supported by AMASS and choose the motion capture dataset and motion sequence that you want to generate a pose from. for now, we only provide support for CMU motion capture dataset *(we're planning to support the rest of the datasets in the near future)*, so you have to choose a pose from the variaty of poses that could be extracted from this dataset.
+	1. for example, after exploring different motion sequences in CMU mocap dataset, we are interested in generating a body pose represents a person kicks a ball. (you can expore all motion sequences categories for CMU dataset through dataset website `http://mocap.cs.cmu.edu/search.php?subjectnumber=%&motion=%`. go to website and select all motion tab to see differnt motions)
+	2. from differnet motion categories there, we are interested in kick soccer ball category (note it is listed as subject # 10, we need this number among other identifiers to identify the pose we are interested in.)
+6. From AMASS download page `https://amass.is.tue.mpg.de/dataset`, many differnt motion capture dataset are listed, choose the one you are interested to generate a pose from and download Body data, for example we choose CMU motion capture dataset to download.
+	<img src="instructions_screenshots/mocap_dataset_download.png"> 
+	download the dataset and extract it under `resources/AMASS` folder.
+
+
+## Usage
+Execute in the BlenderProc main directory:
+
+```
+python run.py examples/amass_human_poses/config.yaml resources/AMASS examples/amass_human_poses/output
+```
+
+* `examples/amass_human_poses/config.yaml`: path to the configuration file with pipeline configuration.
+
+## Visualization
+
+In the output folder you will find multiple `.hdf5` files. These can be visualized with the script:
+
+```
+python scripts/visHdf5Files.py examples/amass_human_poses/output/*.hdf5
+```
+
+## Steps
+* AMASSLoader first check the taxonomy.json file for the currently supported datasets
+* loads the body parameters for the selected pose from dataset
+* Loads the parametric body model from the downloader body models
```

Comment:
> the downloader body models

What is the downloader body models? Do you mean the downloaded body models?

Reply:
Sorry, it is a typo. I mean downloaded body models